### PR TITLE
Jagadeesh_Assignment_28_25min

### PR DESCRIPTION
**Pull request by JagadeeshMJun20**
Please review my code.
Output:
image

### CODE REVIEW

File: src/jagadeeshMJun20/SumOfNumbers.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Input: J7u9y8h1h8j4b7j3j6
+Ouput: Sum of all numbers in String is - 54. */
+package jagadeeshMJun20;
+
+public class SumOfNumbers {
+	void displaySum(String inputStr) {
+		int sum = 0;
+		for (int index = 0; index < inputStr.length(); index++) {
+			int x = inputStr.charAt(index);
+			if (x > 47 && x < 58) {
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/SumOfNumbers.java:13]: Here the condition should be x>=47 && x<=58. It should include both the values.

File: src/jagadeeshMJun20/SumOfNumbers.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+				sum = sum + Character.getNumericValue(x);
+			}
+		}
+		System.out.println("Sum of all numbers in String is - "+sum);
+
```
Comment (MadhuraMJun20) [inline src/jagadeeshMJun20/SumOfNumbers.java:18]: Remove extra spaces.

**Review by MadhuraMJun20 — CHANGES_REQUESTED** [review]
Please make the changes.

**Review by MadhuraMJun20 — APPROVED** [review]
Code looks good to me.

File: src/jagadeeshMJun20/SumOfNumbers.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Ouput: Sum of all numbers in String is - 54. */
+package jagadeeshMJun20;
+
+public class SumOfNumbers {
+	void displaySum(String inputStr) {
+		int sum = 0;
+		for (int index = 0; index < inputStr.length(); index++) {
+			int x = inputStr.charAt(index);
+			if (x > 47 && x < 58) {
+				sum = sum + Character.getNumericValue(x);
```
Comment (KrishnaKTechnocredits) [inline src/jagadeeshMJun20/SumOfNumbers.java:14]: sum += (str.charAt(index) - '0'); You can use this as well.

**Pull request merged**